tests/test_model.py

```diff
@@ -135,3 +141,49 @@ def test_optional_link(self):
         # should throw "Please provide a link." error
         with self.assertRaises(ValidationError):
             plugin2.clean()
+
+    @skipUnless(CMS_35, "Test relevant only for CMS>=3.5")
+    def test_in_placeholders(self):
```

Comment:
the same issues from PR 172 apply here as well. test isolation is noch given due to the created objects